backend/fms_core/template_importer/importers/_generic.py

```diff
@@ -16,12 +23,16 @@ def __init__(self):
 
         self.preloaded_data = {}
         self.file = None
+        self.format = None
+        self.imported_file = None
         self.sheets = {}
         self.previews_info = []
 
-    def import_template(self, file, format, dry_run):
+    def import_template(self, file, dry_run, user = None):
         self.file = file
-        self.format = format
+        file_name, file_format = os.path.splitext(file.name)
```

Comment:
I can't believe the python api uses this typo for a function name. It should be "splittext" or "split_text". :)

Reply:
I think it is split_ext as in split extension. But yeah it is not that friendly